fluent_contents/plugins/twitterfeed/templates/plugins/twitterfeed/base.html

```diff
@@ -1,51 +1,5 @@
-{% load i18n twitterfeed_tags %}
-{% comment %}
-
-  The twitterfeed is currently rendered fully client-side.
-  To add a server-side rendering, look for django-twitter-tag.
-
-  The script uses jQuery.tweet, which offers quite a lot of features with quite decent configuration
-  (granted not all HTML styles and formatting can be defined.. I suggest forking it if that's needed)
-
-  TODO:
-    Currently the feed scripts are included inline,
-    making it impossible to load multiple instances.
-    This still needs to be implemented in django-fluent-contents,
-    either using django-sekizai or a custom media mechanism.
-
-{% endcomment %}
-<script type="text/javascript" src="{{ STATIC_URL }}fluent_contents/twitterfeed/jquery.tweet.js"></script>
-<script type="text/javascript">
-//<![CDATA[
-(function(){
-
-  function setupFeed()
-  {
-    jQuery('#twitter-container-{{ instance.pk }}').tweet({
-      {% block jsoptions %}
-      username: '{{ instance.twitter_user }}',
-      {% endblock %}
-      {% block base_jsoptions %}
-      avatar_size: {{ AVATAR_SIZE }},
-      refresh_interval: {{ REFRESH_INTERVAL }},
-      template: '{{ TEXT_TEMPLATE|escapejs }}',
-      loading_text: '{% trans "loading tweets..." %}',
-      {% endblock %}
-      count: {{ instance.amount }}
-    });
-  }
-
-  if( window.jQuery )
-    jQuery(document).ready(setupFeed);
-  else
-    setTimeout(setupFeed, 10);
-
-})();
-//]]>
-</script>
-
 <div class="twitterfeed">
-	<h2>{{ instance.title|urlize_twitter }}</h2>
-	<div id="twitter-container-{{ instance.pk }}">{% block htmlcontainer %}{% endblock %}</div>
-	{% if instance.footer_text %}<p>{{ instance.footer_text|urlize_twitter }}</p>{% endif %}
+  {% if instance.title %}<h2>{{ instance.title }}</h2>{% endif %}
+  <div id="twitter-container-{{ instance.pk }}">{% block htmlcontainer %}{% endblock %}</div>
+  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```

Comment:
Instead of using some embedded script, adding the following to the ContentPlugin would be nicer:

```
class FrontendMedia:
    js = ('//platform.twitter.com/widgets.js',)
```